app/frontend/src/router/index.js

```diff
@@ -127,6 +127,20 @@ export default function getRouter(basePath = '/') {
             },
             props: createProps,
           },
+          {
+            path: 'export',
+            name: 'SubmissionsExport',
+            component: () =>
+              import(
+                /* webpackChunkName: "export" */ '@/views/form/Export.vue'
+              ),
+            meta: {
+              breadcrumbTitle: 'Submissions Export',
+              requiresAuth: IdentityProviders.IDIR,
```

Comment:
IDIR only for submissions export? Shouldn't it be just requiresAuth: true?

Reply:
Thanks for this. I mixed Form Designer up with the Submission Export

Reply:
The SubmissionsExport here should be requiresAuth: true

Reply:
This has been fixed